volumes/volume.go

```diff
@@ -113,10 +114,11 @@ func (v *Volume) ToDisk() error {
 }
 
 func (v *Volume) toDisk() error {
-	data, err := json.Marshal(v)
```

Comment:
same here, the way you can remove Marshal, is by looking at what the code is doing with `data`: in this case it's again writing to a file so -> os.Create.